pipeline/pipeline_interface_project.yaml

```diff
@@ -0,0 +1,6 @@
+pipeline_name: count_lines
+pipeline_type: project
+var_templates:
+  pipeline: '{looper.piface_dir}/count_lines.sh'
```

Comment:
can this be simplified by moving this into the command_template, instead of having to use a `var_template` ?

Reply:
this was to make this as simple as possible